ray-on-gke/rayservice-examples/rayservice-tiiuae-falcon-40b-quantized.yaml

```diff
@@ -0,0 +1,109 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: ray.io/v1alpha1
```

Comment:
Can we try to use the v1 API instead of v1alpha1?

Reply:
This would require updating `gke-platform` to kuberay 1.0. So we'd have to do regression on all the other examples. 

So leave for `v1alpha1` for now and come back to this when we rev kuberay?

Reply:
i have updated to kuberay `1.0.0` and updated the CR's to `v1`. no issues after testing.